onnxruntime/core/session/inference_session.cc

```diff
@@ -946,6 +971,46 @@ class InferenceSession::Impl {
     session_state_.SetLogger(*session_logger_);
   }
 
+  // Registers all the predefined transformers with transformer manager
+  void AddPredefinedTransformers(GraphTransformerManager& transformer_manager, const uint32_t& enabled_levels, const std::vector<std::string>& custom_list) {
+    if ((enabled_levels & TransformerLevel::Optional_L1) || !custom_list.empty()) {
+      auto level = TransformerLevel::Optional_L1;
+      // Generate and register rewrite rules for L1
+      auto rewrite_rules_to_register = transformerutils::GenerateRewriteRules(level, custom_list);
+      if (!rewrite_rules_to_register.empty()) {
+        std::unique_ptr<RuleBasedGraphTransformer> graph_rewrite_rules = std::make_unique<TopDownRuleBasedTransformer>("L1_RuleBasedTransformer", "Top down transformer");
```

Comment:
>Top down transformer [](start = 147, length = 20)

Possibly better to describe what this transformer will do (e.g. "apply global L1 rules") than what it is ("Top down transformer").